### PR TITLE
osd: refact include of `activate_osds.yml`

### PR DESCRIPTION
**Pull request by guits**
remove duplicate code.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

File: roles/ceph-osd/tasks/main.yml [hunk 1]
```diff
@@ -41,6 +41,9 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
+- include: ./activate_osds.yml
+  when: not containerized_deployment
```
Comment (leseb) [inline roles/ceph-osd/tasks/main.yml]:
```
when:
  - not conta...
```

thanks!

Comment (guits) [reply roles/ceph-osd/tasks/main.yml]: @leseb done

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-bluestore_docker_dedicated_journal

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-docker_dedicated_journal

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-update_docker_cluster

**Pull request merged**